.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
+  extends: [
+    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    'plugin:you-dont-need-lodash-underscore/compatible', // Try to avoid using lodash in the codebase
+  ],
+  parserOptions: {
+    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+    sourceType: 'module', // Allows for the use of imports
+  },
+  rules: {
+    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
+    quotes: ['error', 'single'],
+    'no-unused-vars': 'error',
+    '@typescript-eslint/interface-name-prefix': [
+      'error',
+      {
+        prefixWithI: 'never',
+      },
+    ],
+  },
+  'overrides': [
+    {
+      'files': ['src/locales/*.ts'],
+      'rules': {
```

Comment:
I’ve just noticed some `prettier` errors here. We should remove the single quotes from the key names on L25-28.

I could create a PR for this, however, this is such a small change that @emagnier could fix this when he find the required time to merge this PR.